spec/helpers.lua

```diff
@@ -269,7 +352,10 @@ local pack = function(...) return { n = select("#", ...), ... } end
 local unpack = function(t) return unpack(t, 1, t.n) end
 
 --- Prints all returned parameters.
--- Simple debugging aid.
+-- Simple debugging aid, it will pass all received parameters, hence will not
+-- influence the flow of the code. See also `fail`.
```

Comment:
That section is already there in `@see fail`
```suggestion
-- influence the flow of the code.
```

Reply:
This is more explicit imo, lmk if you want me to remove it